app/src/dataAnalysis.py

```diff
@@ -0,0 +1,75 @@
+import json
+import datetime
+import operator
+import re
+import copy
+
+class memberObj(object):
+    pageUrl = ""
+    name = ""
+    facebookId = ""
+    amount = 0
+    isTeamCaptain = False
+    supportor = {}
+
+    def __init__(self, pageUrl, name, facebookId,amount, isTeamCaptain):
+        self.pageUrl = pageUrl
+        self.name = name
+        self.facebookId = facebookId
+        self.amount = amount
+        self.isTeamCaptain = isTeamCaptain
+
+#today = datetime.date.today()
+today = datetime.date(2017,9,5)
+completeData = {}
+
+def importData(filePath):
+    with open(filePath) as json_file:
+        jsonData = json.load(json_file)
+        
+    for date in jsonData:
+        dayData = []
+        d = date.split(sep=',')
+        d = [int(x) for x in d]
+        datestamp = datetime.date(d[0],d[1],d[2])
+        for member in jsonData[date]:
+            amount = float(re.sub('[$]','',member["amount"]))
+            tempVar = memberObj(member["pageUrl"], member["name"], member["facebookId"], amount, member["isTeamCaptain"])
+            dayData.append(tempVar)
+        completeData[datestamp] = dayData
+
+importData("data.json")
+
+def highestDonor(Dict):
+    topDonor = Dict[0]
+    for donor in Dict:

```

Comment:
try this instead of a for loop to make it faster: 
top_donor = max(Dict, key=...)